wiz/README.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Wiz Buildkite Plugin Template
```

Comment:
I'd suggest removing the "plugin" part of the title and lean more into the specific use-case. The plugin part is more of an implementation detail that can be expanded on in the readme.

e.g. 
```suggestion
title: Scan for vulnerabilities using Wiz
```

Reply:
done